ui/src/components/Issuer/IssuedToken.scss

```diff
@@ -1,83 +1,104 @@
-
-.ui.table.issuer-cap-table tr td {
-    border-top: 0px !important;
-}
-
-.ui.table.issuer-cap-table tr:nth-child(odd) {
-    background-color: var(--grey3);
-}
-
-.ui.table.issuer-cap-table tr th {
-    background-color: var(--white);  
-    border-bottom: 0px !important;
-}
-
-.ui.table.issuer-cap-table tr th:nth-of-type(1){
-    padding-left: 0px;
-}
-
-.ui.table.issuer-cap-table {
-    border: 0px !important;
-}
-
-.token-subheading {
-    display: flex;
-    flex-direction: row;
-    justify-content: space-between;
-    align-items: center;
-}
-
-.token-details {
-    display: flex;
-    flex-direction: row;
-}
-
-.token-details p {
-    margin-left: var(--spacing-m);
-}
-
-.participants-viewer {
-    padding: var(--spacing-m) 0 var(--spacing-m) 0;
-    border-bottom: 2px solid var(--grey3)
-}
-
-p.toggle-participants:hover {
-    cursor: pointer;
-}
-
-.list-heading {
-    display: flex;
-    flex-direction: row;
-    padding: var(--spacing-m) 0 var(--spacing-s) 0;
-    align-items: center;
-}
-
-.list-heading button.ghost {
-    margin-left: var(--spacing-m);
-}
-
-ul.participants-list  {
-    display: inline-flex;
-    flex-direction: column;
-    max-height: 128px;
-    flex-wrap: wrap;
-    list-style-type: none;
-    padding-left: 0px;
-    width: unset;
-    margin: 0px;
-}
-
-ul.participants-list li {
-    padding: var(--spacing-s) var(--spacing-l) 0 0;
-    width: fit-content;
-}
-
-@media screen and (max-width: 1000px) {
-    .ui.table.issuer-cap-table tr th, .ui.table.issuer-cap-table tr td {
-        text-align: left;
-    }
-
-    .ui.table.issuer-cap-table tr:nth-child(odd) {
-        background-color: unset;
+@import "../common/DonutChart.scss";
+
+@mixin issued-token {
+    .issued-token {
+        @include donut-chart;
+
+        .position-holdings-data {
+            display: flex;
+            flex-direction: row;
+
+            .ui.table.issuer-cap-table {
+                border: 0px !important;
```

Comment:
`!important` makes me sad :'( 

Eventually I'd like to try to override the built-in semantic UI styles by applying higher specificity ([this](https://uxengineer.com/css-specificity-avoid-important-css/) is a neat article about it) through classes/ids/attributes/nesting rather than important 

That'll probably be a chunk of work though, so the `important` can stay for now